spec/factories/seeds/teacher_profile_factory.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_teacher_profile, class: "TeacherProfile") do
+    trn { Faker::Number.unique.rand_in_range(10_000, 100_000).to_s }
+
+    trait :with_school do
+      school { build(:seed_school) }
+    end
+
+    trait :with_participant_identity do
+      participant_identity { build(:seed_participant_identity) }
+    end
```

Comment:
Sort of feels wrong to have this in here as it's not directly associated with a :teacher_profile?

Reply:
I think that might've been some over enthusiastic copy/pasting!